### ISSUE TITLE
Test error with Go 1.14beta1

### ISSUE THREAD

**Issue by eclipseo** [circonus-labs/circonus-gometrics#107]
Fedora Rawhide x86_64, with Go 1.14beta1, and circonus-gometrics 3.0.0 beta4:

```
Testing    in: /builddir/build/BUILD/circonus-gometrics-c6f494a5b139fc62376f353711f5e4abce77f2b1/_build/src
         PATH: /builddir/build/BUILD/circonus-gometrics-c6f494a5b139fc62376f353711f5e4abce77f2b1/_build/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/sbin
       GOPATH: /builddir/build/BUILD/circonus-gometrics-c6f494a5b139fc62376f353711f5e4abce77f2b1/_build:/usr/share/gocode
  GO111MODULE: off
      command: go test -buildmode pie -compiler gc -ldflags "-X github.com/circonus-labs/circonus-gometrics/version=3.0.0 -X github.com/circonus-labs/circonus-gometrics/version.commit=c6f494a5b139fc62376f353711f5e4abce77f2b1 -extldflags '-Wl,-z,relro -Wl,--as-needed  -Wl,-z,now -specs=/usr/lib/rpm/redhat/redhat-hardened-ld '"
      testing: github.com/circonus-labs/circonus-gometrics
github.com/circonus-labs/circonus-gometrics
PASS
ok  	github.com/circonus-labs/circonus-gometrics	0.015s
github.com/circonus-labs/circonus-gometrics/api
2020/01/28 23:50:32 circonus-gometrics/api is DEPRECATED and will be removed - switch to github.com/circonus-labs/go-apiclient
--- FAIL: TestNew (0.00s)
    api_test.go:70: invalid config [nil]
    api_test.go:82: invalid config [blank]
    api_test.go:95: API Token, no API App, no API URL
    api_test.go:106: API Token, API App, no API URL
    api_test.go:118: API Token, API App, API Account ID, no API URL
    api_test.go:131: API Token, API App, API URL [host]
    api_test.go:144: API Token, API App, API URL [trailing '/']
    api_test.go:157: API Token, API App, API URL [w/o trailing '/']
    api_test.go:170: API Token, API App, API URL [invalid]
    api_test.go:183: Expected an '&errors.errorString{s:"parse http://something.somewhere.com\\somepath$: invalid character \"\\\\\" in host name"}' error, got '&url.Error{Op:"parse", URL:"http://something.somewhere.com\\somepath$", Err:"\\"}'
    api_test.go:187: Debug true, no log.Logger
FAIL
exit status 1
FAIL	github.com/circonus-labs/circonus-gometrics/api	29.162s
```

**Comment by maier** [comment]
please try the latest release v3.0.0

**Comment by eclipseo** [comment]
It is indeed fixed with 3.0.0. Thanks!
